ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/documentation/fabric/TWODC_5STAGE_CLOS-documentation.md

```diff
@@ -245,31 +245,20 @@
 
 | Key | Type | Description |
 | --- | ---- | ----------- |
-| access_points | access_point | Access Point |
-| cameras | camera | Camera |
-| cpes | cpe | CPE |
-| firewalls | firewall | Firewall |
-| generic_devices | generic_device | Generic Device |
-| load_balancers | load_balancer | Load Balancer |
-| phones | phone | Phone |
-| printers | printer | Printer |
-| routers | router | Router |
 | servers | server | Server |
-| storage_arrays | storage_array | Storage Array |
-| workstations | workstation | Workstation |
```

Comment:
is this one exepected? Changed since my last review

Reply:
We now only show the types that are actually in use.